frontend/src/Redux/actions.ts

```diff
@@ -125,7 +125,7 @@ export const fetchAssistantsData = () => {
     const project_base_url = 'api/v1'
 
     dispatch(fetchAssistantRequest());
-    request.get(`${project_base_url}/assistants?limit=20`)
+    request.get(`${project_base_url}/ui/assistants?limit=20`)
```

Comment:
NIT: Should have all requests wrapped under `/axios` , we should not see requests being used directly at other places. Not blocking this PR to be merged though